react/features/conference/middleware.web.ts

```diff
@@ -19,6 +20,8 @@ MiddlewareRegistry.register(store => next => action => {
                 /* onAllow */ () => APP.store.dispatch(setCameraFacingMode(data.facingMode)),
                 /* initiatorId */ participant.getId()
             ));
+        } else if (data?.name === LOWER_HAND && data?.isModerator) {
```

Comment:
Don't check the payload, since that could be set to whatever, check that the sender (the `participant` variable) is a moderator.